src/log-receiver/log-receiver.service.ts

```diff
@@ -32,19 +37,38 @@ export class LogReceiverService implements OnModuleInit {
     private http: HttpService,
     private configService: ConfigService,
     @InjectModel('logs') private logModel: Model<Logs>,
+    private serviceRegistration: ServiceRegistrationService,
   ) {
     // Create an Issue Creator for each LogType
-    this.cpuUtilizationIssueCreator = new CpuUtilizationIssueCreatorComponent(http, configService);
-    this.timeoutIssueCreator = new TimeoutIssueCreatorComponent(http, configService);
-    this.cbOpenIssueCreator = new CbOpenIssueCreatorComponent(http, configService);
-    this.errorResponseIssueCreator = new ErrorResponseIssueCreatorComponent(http, configService);
-    this.kafkaUrl = this.configService.get<string>('KAFKA_URL', 'localhost:9092');
-    this.kafka = new Kafka({
+    this.cpuUtilizationIssueCreator = new CpuUtilizationIssueCreatorComponent(
+      http,
+      this.logModel,
+      configService,
+    );
+    this.timeoutIssueCreator = new TimeoutIssueCreatorComponent(
+      http,
+      this.logModel,
+      configService,
+    );
+    this.cbOpenIssueCreator = new CbOpenIssueCreatorComponent(
+      http,
+      this.logModel,
+      configService,
+    );
+    this.errorResponseIssueCreator = new ErrorResponseIssueCreatorComponent(
+      http,
+      configService,
+      this.logModel
```

Comment:
sicher, dass die argumente in der richtigen reihenfolge sind? ich denke, es sollte erst logmodel, dann configservice sein.

Reply:
Das sollte passen, da wir TypeScript benutzen wird ja auch ein Fehler angezeigt wenn die Typen nicht stimmen würden